src/devices/src/virtio/block/request.rs

```diff
@@ -656,22 +656,26 @@ mod tests {
         // We will check that Request::parse() arrives at the same result after
         // parsing the descriptor chain. Input properties are validated and commented below.
         (
-            any::<u64>(), // random data buffer sparsity factor
-            any::<u32>(), // data_len
-            any::<u64>(), // sector
-            any::<RequestType>(),
+            any::<u64>(),         // random data buffer sparsity factor
+            any::<u32>(),         // data_len
+            any::<u64>(),         // sector
+            any::<RequestType>(), // request type
+            any::<[bool; 32]>(),  // coin
```

Comment:
why 32 coins? AFAICT 5 coins are used

Reply:
True. I would like to have a margin of error anyway. I reduced the number of coins to 10.